project/project.go

```diff
@@ -14,13 +14,20 @@ import (
 )
 
 const (
-	Status200 int = 200
 	Status201 int = 201
 	Status400 int = 400
 	Status401 int = 401
 	Status403 int = 403
 	Status404 int = 404
+	Status409 int = 409
```

Comment:
Why do we even have these constants in the first place? It's not like we'd redefine `Status409 = 410` anytime soon. On a side note: Why not use the [status code constants from the `http` package](https://golang.org/pkg/net/http/#pkg-constants)?